packages/manager/cypress/e2e/core/linodes/search-linodes.spec.ts

```diff
@@ -39,7 +34,24 @@ describe('Search Linodes', () => {
 
       // Use the main search bar to search and filter linode by id: pattern
       ui.mainSearch.find().clear().type(`id:${linode.id}`);
+
+      // Verify the Linode shows as an option in the main search autocomplete
       ui.autocompletePopper.findByTitle(linode.label).should('be.visible');
+
+      // Press Enter to go to the search page
+      cy.focused().type('{enter}');
+
+      // Verify the search field still has the correct search query that the user typed
+      ui.mainSearch
+        .find()
+        .findByRole('combobox')
+        .should('have.value', `id:${linode.id}`);
+
+      // Verify we land on the search page
+      cy.url().should('endWith', '/search?query=tag%3Alke');
```

Comment:
test failure's just caused by a small typo:

```suggestion
      cy.url().should('endWith', `/search?query=id%3A${linode.id}`);
```

Reply:
oops outdated!

Reply:
Thanks! Not sure what I was doing with `?query=tag%3Alke` haha